src/server.ts

```diff
@@ -1,25 +1,34 @@
 import * as express from 'express';
 import { URL } from 'url';
-import * as path from 'path'
+import * as path from 'path';
 
 import * as meetbotManager from './meetbot-manager';
 
 const HTTP_PORT = process.env.HTTP_PORT || 8080;
 const server = express();
 
 server.use(express.json());
-server.use('/ui', express.static(path.join(__dirname, './public')))
+server.use('/ui', express.static(path.join(__dirname, './public')));
 
 server.post('/join', async (req, res) => {
 	let meetMetadata;
 	// Validate body
 	if (!req.body.url) {
-		return res.status(400).send('Missing `url` value in payload body.');
+		return res
+			.status(400)
+			.send('Unable to join. Missing `url` value in payload body.');
 	}
 	try {
 		meetMetadata = new URL(req.body.url);
+		if (meetMetadata.hostname !== 'meet.google.com') {
+			return res
+				.status(400)
+				.send('Unable to join. Invalid Google Meet URL in payload body. \n');
```

Comment:
could throw an error to use the 400 response in the catch. removes duplicate code

Reply:
Sounds good on both suggestions. Implementing.